main.go

```diff
@@ -194,8 +194,7 @@ func mountWithArgs(
 			conn, err = getConnWithRetry(flags)
 		}
 		if err != nil {
-			mountStatus.Printf("Failed to open connection: %v\n", err)
```

Comment:
by removing this you are removign the log from the file. We can't rely on the console logs.

Reply:
I removed from here, because this is the only place where we are printing the error. Other than this, we just return the error.

Reply:
Done, as discussed added a global print statement to preserve the error-logs in a log-file/syslog.